manifests/charts/ztunnel/templates/_helpers.tpl

```diff
@@ -0,0 +1,10 @@
+{{- define "istio-labels" }}
+    app.kubernetes.io/name: ztunnel
```

Comment:
In the future, `name` itself should be templated, so this can be reused.

And we can probably include the equivalent of `{{ with .Values.labels -}}{{ toYaml . | nindent 4}}{{ end }}` in the template itself as well.